matching-notification-migrations/src/main/java/org/phenotips/matchingnotification/internal/R74693PatientNetwork377DataMigration.java

```diff
@@ -89,9 +81,19 @@ public void hibernateMigrate() throws DataMigrationException, XWikiException
         Transaction t = session.beginTransaction();
 
         try {
-            Query q1 = session.createQuery("update " + PatientMatch.class.getCanonicalName()
-                + " set comments = concat('[{ comment:', comment, '}]') where comment <> ''");
-            q1.executeUpdate();
+            Query q = session.createQuery("from " + PatientMatch.class.getCanonicalName() + " where comment <> ''");
```

Comment:
This query will break when `comment` field is un-mapped from the PatientMatch class. Also this query will likely break in `1.3` since there will be multiple implementations of the `PatientMatch` interface and not sure which table the query will use then (need to specify that explicitly here I think). We discussed this with Veronika already, and will make changes